warp/tests/interop/test_jax.py

```
@@ -20,33 +20,33 @@
 
 # basic kernel with one input and output
 @wp.kernel
-def triple_kernel(input: wp.array(dtype=float), output: wp.array(dtype=float)):
+def triple_kernel(input: wp.array[float], output: wp.array[float]):
```

Ruff (0.15.12)

[error] Function argument `input` is shadowing a Python builtin

(A002)